### PR TITLE
2.4.2 Release

### PR DESCRIPTION
**Pull request by outofambit**
- [x] finalize release notes
- [x] release

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "2.4.2": [
+      "[Fixed] Prevent autocomplete suggestions from overlapping in some cases - #9474",
```
Comment (billygriffin) [inline changelog.json:4]:
```suggestion
      "[Fixed] Prevent autocomplete suggestions from overlapping when loading many results - #9474",
```

**Comment by outofambit** [comment]
adding the followup beta release to this PR now

**Comment by outofambit** [comment]
we shipped these!

**Comment by rafeca** [comment]
Merging to get these changes into development!

**Pull request merged**